src/modules/simulator/simulator.h

```diff
@@ -330,20 +330,21 @@ class Simulator : public ModuleParams
 
 	mavlink_hil_actuator_controls_t actuator_controls_from_outputs(const actuator_outputs_s &actuators);
 
-	void handle_message(mavlink_message_t *msg, bool publish);
+	void handle_message(mavlink_message_t *msg);
 	void handle_message_distance_sensor(const mavlink_message_t *msg);
 	void handle_message_hil_state_quaternion(const mavlink_message_t *msg);
 	void handle_message_landing_target(const mavlink_message_t *msg);
 	void handle_message_optical_flow(const mavlink_message_t *msg);
 
 	void parameters_update(bool force);
 	void poll_topics();
-	void poll_for_MAVLink_messages(bool publish);
+	void poll_for_MAVLink_messages();
 	void request_hil_state_quaternion();
 	void send();
 	void send_controls();
 	void send_heartbeat();
 	void send_mavlink_message(const mavlink_message_t &aMsg);
+	void set_publish(const bool publish = false);
```

Comment:
This default is confusing to me: if I were to see a `simulator->set_publish()`, I would not expect it to disable publication.

Reply:
No worries, I can set it to default true.

Reply:
Done!

Reply:
Good to go?